.yamato/_run-all.yml

```diff
@@ -63,6 +85,22 @@ all_package_tests:
 {% endfor -%}
 {% endfor -%}
 
+# Test minimal project with different versions of dependencies
+all_compatibility_tests:
```

Comment:
Handy Run All just for these, incase we update UTP in the future and just want to kick off these builds.

Reply:
<img width="720" alt="image" src="https://user-images.githubusercontent.com/36935028/189459883-0959ada3-ab29-49fd-98e9-184844180d3a.png">